src/libs/actions/Report.js

```diff
@@ -1155,7 +1141,6 @@ function readNewestAction(reportID) {
     API.write('ReadNewestAction',
         {
             reportID,
-            sequenceNumber,
```

Comment:
I think this should not be removed?

Reply:
I don't remeber removing this, maybe it was by accident 🤔